app/presenters/summary/components/funding_decision.rb

```diff
@@ -0,0 +1,64 @@
+module Summary
+  module Components
+    class FundingDecision < BaseRecord
+      private
+
+      def answers # rubocop:disable Metrics/AbcSize,Metrics/MethodLength,Metrics/CyclomaticComplexity,Metrics/PerceivedComplexity
+        [
+          Components::FreeTextAnswer.new(
+            :funding_decision_maat_id, funding_decision.maat_id,
+            show: funding_decision.maat_id.present?
+          ),
+          Components::FreeTextAnswer.new(
+            :funding_decision_case_number, funding_decision.reference,
+            show: funding_decision.reference.present?
+          ),
+          Components::ValueAnswer.new(
+            :funding_decision_ioj_result, funding_decision.interests_of_justice&.result,
+            show: funding_decision.interests_of_justice.present?
```

Comment:
Same here (you're using the safe navigation operator to set the value, so should be fine). 

We know that any decision shown here is valid and complete and therefore fine to ignore any rows not given.